build.yaml

```diff
@@ -317,6 +348,25 @@ common_cli:
     # and build a rpm/deb package using docker
     parse_logs \$venv/bin/python admin/build-package --destination-path repo --distribution \${DISTRIBUTION_NAME} /flocker/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
 
+  build_homebrew_package: &build_homebrew_package  |
+    # and build a homebrew
```

Comment:
Expand on this, we are building a source distribution and then a Homebrew recipe which installs that source distribution and its dependencies.